src/config/config.service.ts

```diff
@@ -83,6 +83,10 @@ export class ConfigService {
     return this.nestConfigService.get<number>('QUEUE_HIGH_WATER')!;
   }
 
+  public getCapacityBatchLimit(): string {
+    return this.nestConfigService.get<string>('CAPACITY_BATCH_LIMIT')!;
+  }
+
```

Comment:
If you're adding a new environment variable, you should also add it to the interface at the top of this file, as well as add a validation to `env.config.ts`